README.rst

```diff
@@ -122,6 +122,18 @@ Install spatialite and sqlite:
                             gdal-bin libproj-dev libgeos-dev libspatialite-dev \
                             fping
 
+Quickstart
+----------
+
+1. Setting openwisp-config management_interface option
```

Comment:
make subheadings instead of lists, always flag code variables with double slant quotes.